aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/parser/InteractionTypeParserTest.xtend

```diff
@@ -122,4 +122,22 @@ class InteractionTypeParserTest extends AbstractParserTest {
 		interactionType.assertNoSyntaxErrors
 	}
 
+	@Test
+	def void parseWithLocatorStrategy() {
+		// Given
+		val input = '''
+			interaction type MyAddition {
+				template = "Add" ${x} "and" ${y} "and write into" ${element} "."
+				method = MyFixture.addAndWrite(x, y, element)
+				locatorStrategy = ID
+			}
+		'''
+
+		// When
+		val interactionType = input.parse(InteractionType)
+
+		// Then
+		interactionType.assertNoSyntaxErrors
```

Comment:
Since this is only a partial parse (without import of the respective enumeration) the parsed locatory strategy stays unresolved (as proxy)